docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -407,7 +407,7 @@ Now, to add state to a link, pass it via a `state` prop.
 ```jsx
 const NewsFeed = () => (
   <div>
-    <Link to="photos/123" state={{ fromNewsFeed: true }} />
+    <Link to="photos/123" state={{ fromFeed: true }} />
```

Comment:
The example was mixing `fromNewsFeed` and `fromFeed`, but this change makes it consistent.

![image](https://user-images.githubusercontent.com/2344137/45073193-15f72680-b093-11e8-918b-db8d138031eb.png)